test/functional/fixtures/api/es-next/selector/test.js

```diff
@@ -25,7 +25,32 @@ describe('[API] Selector', function () {
         return runTests('./testcafe-fixtures/selector-test.js', 'String ctor argument');
     });
 
+    it('Should wait for element to appear in DOM', function () {
+        return runTests('./testcafe-fixtures/selector-test.js', 'Wait for element in DOM', { elementAvailabilityTimeout: 2500 });
+    });
+
+    it('Should return `null` or `undefined` if element does not appear within given time', function () {
```

Comment:
https://github.com/DevExpress/testcafe/pull/598/files#diff-f9f4a656b17573b16cd155de3e706528R205
we expect exactly `null`